<launch>
    <!-- Launch the image_processor_node -->
    <node
        pkg="cotracker_pkg"
        type="cotracker_node.py"
        name="cotracker_service"
        output="log">
        <rosparam file="$(find cotracker_pkg)/config/config.yaml" command="load" />
    </node>
</launch>
